packages/psss-server/src/connections/AuthConnection.ts

```diff
@@ -52,6 +52,6 @@ export class AuthConnection extends ApiConnection {
       throw new Error('Invalid response from auth server');
     }
     const { accessPolicy, email } = user;
-    return { accessToken, refreshToken, accessPolicy, email };
+    return { accessToken, refreshToken, accessPolicy, email, user };
```

Comment:
The front end needs the user object to display the users name (and less importantly the users other details for the profile section). It may be better to restructure the response rather than just adding in the extra object. If we do that then the 'types' need to be refactored too.